tdrs-backend/tdpservice/search_indexes/reparse.py

```diff
@@ -0,0 +1,110 @@
+"""Reparsing command for selected files."""
+# should include all the steps in the management command
+from tdpservice.data_files.models import DataFile
+from tdpservice.search_indexes.models.reparse_meta import ReparseMeta
+from tdpservice.core.utils import log
+from tdpservice.users.models import User
+from tdpservice.search_indexes.utils import (
+    backup,
+    get_log_context,
+    assert_sequential_execution,
+    should_exit,
+    handle_elastic,
+    delete_associated_models,
+    count_total_num_records,
+    calculate_timeout,
+    handle_datafiles,
+)
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def clean_reparse(selected_file_ids):
+    """Reparse selected files."""
+    selected_files = [int(file_id) for file_id in selected_file_ids[0].split(",")]
+
+    ######
+    files = DataFile.objects.filter(id__in=selected_files)
+    backup_file_name = "/tmp/reparsing_backup"
+    backup_file_name += "_selected_files"
```

Comment:
Should this be removed since we won't put all the file IDs or other file meta in the backup name?